.kitchen.yml

```diff
@@ -12,8 +12,6 @@ verifier:
 platforms:
   - name: centos-6.9
```

Comment:
These platforms can be just the major versions now. Take a look at postgresql for the format 👍